src/onegov/api/path.py

```diff
@@ -28,7 +29,8 @@ def get_api_endpoint(
     endpoint: str,
     page: int = 0,
     extra_parameters: dict[str, Any] | None = None,
-) -> ApiEndpoint | AuthEndpoint:
+) -> ApiEndpoint[_M] | AuthEndpoint | Any:
```

Comment:
This should be `ApiEndpoint[Any]` as well. Unbound type parameters sometimes make sense when defining callback protocols, but generally they should be avoided.